components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m

```diff
@@ -225,6 +225,12 @@ - (void)testSnackbarOverlayViewWithHighElevation {
                                                         animated:NO
                                                       completion:nil];
 
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
```

Comment:
Tip: Add a comment here explaining that there was some flakiness when running Bazel/Xcode versions of the test and that's why a runloop drain was added.

Reply:
Done